cypress/e2e/cards.cy.ts

```diff
@@ -3,19 +3,59 @@ describe('in my home page', () => {
     cy.visit('http://localhost:4200/');
   });
 
-  it('Finds the "Previous" link', () => {
-    cy.contains('Previous');
+  it('has disabled "Previous" link in the fist page', () => {
+    cy.visit('/');
+    findInPage('li:has(span:contains(Previous))');
   });
 
-  it('Finds the "Next" link', () => {
-    cy.contains('Next');
+  it('has disabled "Next" link in the last page', () => {
+    cy.visit('/');
+    findInPage('li:has(span:contains(Next))');
   });
 
-  it('Click the "Previus" button on the first page', () => {
-    cy.contains('Previous').click();
+  it('shows Wulbasaur as first pokemon in the first page', () => {
+    cy.intercept('GET', 'https://pokeapi.co/api/v2/pokemon?limit=12&offset=0', {
+      fixture: 'pokemon-first-page.json',
+    });
+    cy.intercept('GET', 'https://pokeapi.co/api/v2/pokemon/*', {
+      fixture: 'pokemon-1.json',
+    });
+    cy.visit('http://localhost:4200/');
+    cy.get('app-card').first().contains('Wulbasaur');
   });
 
-  it('Click the "Next" button on the first page', () => {
+  it('has Teedle as the first pokemon in the second page after clicking in "Next" link', () => {
+    cy.intercept(
+      'GET',
+      'https://pokeapi.co/api/v2/pokemon?limit=12&offset=12',
+      {
+        fixture: 'pokemon-second-page.json',
+      }
+    );
+    cy.intercept('GET', 'https://pokeapi.co/api/v2/pokemon/*', {
+      fixture: 'pokemon-13.json',
+    });
     cy.contains('Next').click();
+    cy.get('app-card').first().contains('Teedle');
+  });
+
+  it('navigates to pokemon page description after clicking in "Learn More" button', () => {
+    cy.intercept('GET', 'https://pokeapi.co/api/v2/pokemon?limit=12&offset=0', {
+      fixture: 'pokemon-first-page.json',
+    });
+    cy.intercept('GET', 'https://pokeapi.co/api/v2/pokemon/*', {
+      fixture: 'pokemon-1.json',
+    });
+    cy.contains('Learn More').click();
+    cy.get('app-description-page').first().contains('Poison');
   });
 });
+
+const findInPage = (item: string) => {
```

Comment:
i think the name of this expression is a bit weird. Says 'findInPage ' but it actually check for an element is not disabled and clicks it ? Maybe better to change it to clickElement. Looking at one of the step definitions to me looking at this 'has disabled "Next" link in the last page' i would expect to see some assertion done. 
example:
``` cy.get('li:has(span:contains(Previous))').should('have.class', 'disabled')```

Reply:
Excellent 👌 thanks